src/standalone/Vegetation/Canopy.jl

```diff
@@ -510,7 +525,7 @@ function canopy_surface_fluxes(
     canopy_conductance =
         upscale_leaf_conductance.(
             leaf_conductance,
-            model.parameters.LAI,
+            p.canopy.hydraulics.area_index[:leaf],
```

Comment:
a bit random place for this comment, but following the discussion from yesterday, I agree that it makes sense to have things (prescribed pool like LAI or prescribed parameter like albedo) varying in time in p. 

Y is for PDE... but p can store those other things we get from functions or files. 

Anyway, just saying I agree with this design choice!